src/Reflection/ReflectionClass.php

```
@@ -490,6 +497,56 @@ public function hasConstant(string $name) : bool
     }
 
     /**
+     * Get the reflection object of the specified class constant.
+     *
+     * Returns null if not specified.
+     *
+     * @param string $name
+     * @return ReflectionClassConstant|null
+     */
+    public function getReflectionConstant(string $name): ?ReflectionClassConstant
+    {
+        $constants = $this->getReflectionConstants();
+
+        if (!isset($constants[$name])) {
+            return null;
```

I'd rather have an exception here. `null` makes it a silent failure